subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -2848,9 +2850,11 @@ static int lwm2m_exec_handler(struct lwm2m_message *msg)
 	if (ret < 0) {
 		return ret;
 	}
+  
+  payload = (uint8_t *) coap_packet_get_payload(msg->in.in_cpkt, &payload_len);
```

Comment:
No space after cast:
```suggestion
  payload = (uint8_t *)coap_packet_get_payload(msg->in.in_cpkt, &payload_len);
```